### ISSUE TITLE
nx init: options not found in the documentation.

### ISSUE THREAD

**Issue by Sygnious** [nrwl/nx#16251]
### Documentation issue

- [ ] Reporting a typo
- [ ] Reporting a documentation bug
- [x] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?

https://nx.dev/packages/nx/documents/init#usage

### Additional context or description

The documentation page for nx init is lacking the options. I knew from the guide [Migrating from CRA to NX](https://nx.dev/recipes/adopting-nx/migration-cra) that there exists options such as "--integrated" or "--vite=false", but none of them are mentioned in the documentation itself for nx init.

In my case, I needed the "--nxCloud=false" option, as I have reasons for not adding cloud caching to my migration. Since it's not in the documentation, I had to dig around in the [source code](https://github.com/nrwl/nx/blob/master/packages/nx/src/nx-init/react/index.ts) to find the remaining options, and then test it out.

This is **very cumbersome** for those of us who is migrating from CRA (or any other standalone app), when we need to do adjustments to the migration.

I hereby request that the documentation page is filled with the options available.

**Comment by AgentEnder** [comment]
I believe this in-flight PR will address this: https://github.com/nrwl/nx/pull/16245

**Comment by leosvelperez** [comment]
Yeah, the `nx init` documentation will be improved in a couple of PRs. The https://github.com/nrwl/nx/pull/16245 is one part of the improvements, but that one doesn't address the missing options. That would be included in another PR I'll be creating later.
